std-bits/table/src/main/java/org/enso/nativeimage/workarounds/ReplacementToolkit.java

```diff
@@ -0,0 +1,17 @@
+package org.enso.nativeimage.workarounds;
+
+import com.oracle.svm.core.annotate.Alias;
+import com.oracle.svm.core.annotate.TargetClass;
+import com.oracle.svm.core.annotate.Substitute;
+
+@TargetClass(java.awt.Toolkit.class)
+final class ReplacementToolkit {
```

Comment:
Is that all to get rid of dependency on AWT?

Reply:
For our use-cases, yes.